### ISSUE TITLE
Laravel 5.2 - "symfony/finder": "2.8.*|3.0.*"

### ISSUE THREAD

**Issue by yurtesen** [vsch/laravel-translation-manager#33]
Laravel 5.2 framework composer.json has the require line
"symfony/finder": "2.8._|3.0._"
However translation manager has the require line:
 "symfony/finder": "~2.3"
I can see that in an earlier bug report https://github.com/vsch/laravel-translation-manager/issues/23 somehow it looks like the symfony/finder 3.x was working but I am getting a problem.

Do you have to require symfony/finder when it is already required by laravel?
https://github.com/laravel/framework/blob/5.2/composer.json

```
  Problem 1
    - Conclusion: don't install vsch/laravel-translation-manager 2.1.1
    - Conclusion: remove symfony/finder v3.0.6
    - Installation request for vsch/laravel-translation-manager ^2.1 -> satisfiable by vsch/laravel-translation-manager[2.1.0, 2.1.1].
    - Conclusion: don't install symfony/finder v3.0.6
    - vsch/laravel-translation-manager 2.1.0 requires symfony/finder ~2.3 -> satisfiable by symfony/finder[v2.3.0, v2.3.1, v2.3.10, v2.3.11, v2.3.12, v2.3.13, v2.3.14, v2.3.15, v2.3.16, v2.3.17, v2.3.18, v2.3.19, v2.3.2, v2.3.20, v2.3.21, v2.3.22, v2.3.23, v2.3.24, v2.3.25, v2.3.26, v2.3.27, v2.3.28, v2.3.29, v2.3.3, v2.3.30, v2.3.31, v2.3.32, v2.3.33, v2.3.34, v2.3.35, v2.3.36, v2.3.37, v2.3.38, v2.3.39, v2.3.4, v2.3.40, v2.3.41, v2.3.5, v2.3.6, v2.3.7, v2.3.8, v2.3.9, v2.4.0, v2.4.1, v2.4.10, v2.4.2, v2.4.3, v2.4.4, v2.4.5, v2.4.6, v2.4.7, v2.4.8, v2.4.9, v2.5.0, v2.5.1, v2.5.10, v2.5.11, v2.5.12, v2.5.2, v2.5.3, v2.5.4, v2.5.5, v2.5.6, v2.5.7, v2.5.8, v2.5.9, v2.6.0, v2.6.1, v2.6.10, v2.6.11, v2.6.12, v2.6.13, v2.6.2, v2.6.3, v2.6.4, v2.6.5, v2.6.6, v2.6.7, v2.6.8, v2.6.9, v2.7.0, v2.7.1, v2.7.10, v2.7.11, v2.7.12, v2.7.13, v2.7.2, v2.7.3, v2.7.4, v2.7.5, v2.7.6, v2.7.7, v2.7.8, v2.7.9, v2.8.0, v2.8.1, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6].
```

**Comment by vsch** [comment]
@yurtesen, updating it now. Thank you for the heads up.

**Comment by vsch** [comment]
@yurtesen updated. Waiting for Packagist to make the new release available.

**Comment by vsch** [comment]
@yurtesen, I cannot update the packagist version. Getting database errors on the packagist site.

This repo has the changes under tag 2.1.2.

**Comment by vsch** [comment]
@yurtesen, packagist updated with the fix: 2.1.2

**Comment by yurtesen** [comment]
Thanks!
